cabal.project.freeze

```diff
@@ -1153,6 +1152,8 @@ constraints: any.AC-Angle ==1.0,
              any.hourglass ==0.2.12,
              any.hourglass-orphans ==0.1.0.0,
              any.hp2pretty ==0.10,
+             any.hpack ==0.34.5,
+             any.hpack-dhall ==0.5.3,
```

Comment:
why is this added here?  don't mind either way, just surprised.

Reply:
I just ran the script in `tools/convert-to-cabal` to keep `stack.yaml` and  `cabal.project` in sync. I was confused too, but then thought maybe someone added some dependency and forgot to run that script.